hack/lib/util/environment.sh

```diff
@@ -145,12 +145,14 @@ function os::util::environment::setup_tmpdir_vars() {
     HOME="${FAKE_HOME_DIR}"
     export HOME
 
-    # ensure that the directories are clean
-    for target in $( ${USE_SUDO:+sudo} findmnt --output TARGET --list ); do
-        if [[ "${target}" == "${BASETMPDIR}"* ]]; then
-            ${USE_SUDO:+sudo} umount "${target}"
-        fi
-    done
+		if [[ "$(uname)" != "Darwin" ]]; then
```

Comment:
I think you're an entire level of indentation too far in now... 😆 

If this is disabled, and your mounts are not nicely cleaned up, this will be bad UX but apparently this never worked for you on MacOS anyway. I'd still much rather try to fix this instead by using `mount` to find targets inside of `${BASETMPDIR}` and `umount`ing them -- is this reasonable? How do you do this on OSX today?